frontend/src/service/Change.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Err, ErrorService } from './Error.service';
```

Comment:
I wonder whether we should combine this file with change log API. It seems that they can be grouped into a single unit.